pennylane/devices/default_gaussian.py

```diff
@@ -512,7 +512,7 @@ def photon_number(mu, cov, params, hbar=2.0):
     # pylint: disable=unused-argument
     ex = (np.trace(cov) + mu.T @ mu) / (2 * hbar) - 1 / 2
     var = (np.trace(cov @ cov) + 2 * mu.T @ cov @ mu) / (2 * hbar ** 2) - 1 / 4
-    return ex, var
+    return var, ex
```

Comment:
How come these are swapped? :thinking: These should be returned independently to the order of the covariance matrix and the vector of means.